### PR TITLE
fmf: Add hack for stuck chpasswd

### PR DESCRIPTION
**Pull request by marusak**
Also see https://github.com/cockpit-project/bots/commit/c7ea7746b3

This is needed for downstream gating

### CODE REVIEW

**Comment by marusak** [comment]
More like `fml:...` - testing this in downstream now.

**Comment by marusak** [comment]
packit is failing because `semanage: command not found`

**Comment by marusak** [comment]
Firefox and chpasswd hacks worked in downstream :tada: 
Let also make ti green in upstream.

**Comment by marusak** [comment]
oh, komm schon:
```
+ dnf remove -y setroubleshoot-server
Dependencies resolved.
================================================================================
 Package                       Arch    Version          Repository         Size
================================================================================
Removing:
 setroubleshoot-server         x86_64  3.3.24-1.el8     @koji-override-1  1.3 M
Removing unused dependencies:
 policycoreutils-python-utils  noarch  2.9-15.el8       @baseos           138 k
```

File: test/run-verify-host.sh [hunk 1]
```diff
@@ -18,19 +18,11 @@ nproc
...
-        rm -f /etc/yum.repos.d/epel.repo
-        dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
-        sed -i 's/$releasever/8/' /etc/yum.repos.d/epel*.repo
-        dnf config-manager --enable epel
-    fi
-    dnf install -y chromium
-fi
+# install browser; on RHEL, use firefox-nightly as chromium broke CDP on rhel-9
+# Install firefox to pull in all the deps
+dnf -y install firefox
```
Comment (mvollmer) [inline test/run-verify-host.sh:23]: Why do we install firefox from the repos and then install it again from mozilla.org?

Comment (marusak) [reply test/run-verify-host.sh:23]: "Install firefox to pull in all the deps"

We need some libraries to be present on the system - therefore I install firefox from repos so it pulls it in. But we need nightly as that only has CDP.

File: test/run-verify-host.sh [hunk 2]
```diff
@@ -18,31 +18,26 @@ nproc
...
```
Comment (allisonkarlitskaya) [inline test/run-verify-host.sh:23]: That goes better in the `require:` list in `test/verify.fmf`, no?

Comment (marusak) [reply test/run-verify-host.sh:23]: Not really, it is part of a hack. I hope that one day we can do this in some better way, so keeping it at one place feel cleaner to me.

File: test/run-verify-host.sh (hunk 1 above)
Comment (mvollmer) [reply test/run-verify-host.sh:23]: Ah, makes sense!

**Review by mvollmer — APPROVED** [review]

**Pull request merged**